docs/Platforms/cms.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: "Tallwave Guidelines — Platforms — Content Websites"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Platforms]({{ site.root }}/platforms)
+
+# Content Based Websites
+
+On today's Internet, content is king. Being the top result on a search engine (Google) provides a significant advantage in the marketplace. [Bill Gates allegedly](https://www.craigbailey.net/content-is-king-by-bill-gates/) said in 1996 "Content is where I expect much of the real money will be made on the Internet, just as it was in broadcasting."
+
+In that essay he goes on to say how it's not just about putting content on the Internet. Rather, it's about *relevant* and *innovative* content. This can take the form of something as small as a single landing page, or a massive content platform serving hundreds, or thousands of pages.
+
+## Marketing Websites
+
+A good marketing website can do so many things for our clients. It's the first thing their customers see so it:
+
+ * Gives the first impression of brand, UX, and product.
+ * If done properly, moves customers through the sales pipeline.
+ * Serves as a portal to other products we may have built.
+ * Demonstrates out abilities at web design and development.
+ * Provides the customer with analytics and marketing knowledge.
+
+A marketing site by its nature will have fewer moving parts than a full-fledged application. Its purpose is more to present information and drive fewer well-defined actions than require a high degree of input from the user. Thus, these websites do not involve heavy duty frameworks like React or Angular. We do tend to favor building these on WordPress, or plain HTML and CSS files. These are typically built using different libraries, frameworks, pre-processors, package managers, and toolkits:
```

Comment:
Is the meaning here '...well-defined actions that...' or '...well-defined actions, as apposed to those that require a high degree of input from the user.' ?

Reply:
It means more to move them down a sales funnel rather than explore and interact with everything else on the site. We can make that clear.